SPHINXsys/src/shared/common/small_vectors.cpp

```diff
@@ -0,0 +1,217 @@
+/**
+ * @file 	small_vectors.cpp
+ * @author	Luhui Han, Chi ZHang and Xiangyu Hu
+ * @version	0.1
+ */
+
+#include "small_vectors.h"
+//=================================================================================================//
+namespace SPH {
+
+	//=================================================================================================//
+	Vec2d FirstAxisVector(const Vec2d& zero_vector)
```

Comment:
suggest to have unit tests on small vectors.cpp

Reply:
Ok.